lib/queue.js

```diff
@@ -1,33 +1,54 @@
+// last in in first out
 class Queue {
     constructor() {
```

Comment:
⬇️ See comment below

```suggestion
    constructor(bufferSize = 20) {
```